Mobile Release v1.101.0

SiobhyB opened the pull request:
## Description
Release 1.101.0 of the react-native-editor and Gutenberg-Mobile.

For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/6024

SiobhyB committed:
Release script: Update react-native-editor version to 1.101.0

SiobhyB committed:
Release script: Update with changes from 'npm run core preios'

SiobhyB committed:
Update `react-native-editor` CHANGELOG

geriux commented on packages/react-native-editor/CHANGELOG.md:
Just a heads up that the item `[**] Fix iOS Focus loop for RichText components [#53217]` will be shipped in `1.100.2` so once we merge back this PR into `trunk` we'll resolve the conflicts and put it in the right version.

geriux reviewed (approved):
LGTM!

SiobhyB committed:
Remove incorrect CHANGELOG entry

The iOS focus loop will be included in 1.100.2. I've updated the CHANGELOG accordingly.

SiobhyB committed:
Merge branch 'trunk' into rnmobile/release_1.101.0
